### DISCUSSION TITLE
list ALL THE TEAMS a user is in

### DISCUSSION THREAD

**Discussion by FergusMcMenemie-cnic** [community/community#44193] [Enterprise] [answered]
I wish to obtain a list of the github teams a github user is in for my enterprise or at least for an organisation. 

I have been asked to help with the admin of a messy github set up resulting from company mergers. The appear to be many incorrect docs. Notably https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#get-team-membership-for-a-user .... which starts "get a user's membership with a team"

**Comment by FergusMcMenemie-cnic** [comment, marked as answer]
Answering my iown question..... I just stumbled upon
`https://github.com/orgs/<organisation_name>/teams?query=@<github_username>`

**Reply by nikitamakeyev** [reply]
Life saver!

**Comment by C-PET** [comment]
I tried the same query above but only get ALL teams returned.  Not just the teams a specified user belongs to.
Only difference I guess is we are using github enterprise server not github.com
trying to accomplish the same task.  Given a user in the enterprise list all the teams they belong to.
We have a multi-org design.

**Reply by melroy89** [reply]
The only way is using GraphQL I'm afraid, I know it sucks. `query` parameter will get ignored on `api/v3` endpoints.

Something like this:

```graphql
query {
   organization(login: "your-organization") {
      teams(first: 100, userLogins: ["the-username"]) {
          edges {
            node {
              name
              slug
            }
          }
       }
    }
}
```

Replace `your-organization` and `the-username` of course. But you get the idea.
